### DISCUSSION TITLE
Use open price to calculate Bollinger Bands

### DISCUSSION THREAD

**Discussion by pooyan-fekrati** [DaveSkender/Stock.Indicators#660] [Q&A] [answered]
Hi all,

Ia there a way to use open price instead of close price when using the GetBollingerBands method?

**Comment by DaveSkender** [comment]
It's not part of the standard parameter options like we do for SMA and EMA, but you can still do it by converting your quote history to replace Close price with Open price, then run it through.

```csharp
var altQuotes = quotes
    .Select(x => new Quote
    {
        Date = x.Date,
        Close = x.Open
    });

var results = altQuotes.GetBollingerBands(..);
```

**Comment by DaveSkender** [comment, marked as answer]
Starting with v2, you can now choose any [price element or candle variant](https://daveskender.github.io/Stock.Indicators/utilities/#use-alternate-price) to generate Bollinger Bands.  Here's an example of the syntax:

```csharp
// example
var results = quotes
    .Use(CandlePart.Open)
    .GetBollingerBands(..);
```
